inc/namespace.php

```diff
@@ -1995,6 +1997,16 @@ function handle_autosuggest_endpoint() {
 		],
 	];
 
+	
+	/**
+	 * Filter the autosuggest query JSON.
+	 *
+	 * This is the data that is json encoded and passed as the elasticsearch request body.
+	 *
+	 * @param array $json ES Query.
+	 */
+	$json = apply_filters( 'altis.search.autosuggest_json', $json );
```

Comment:
Only thing I'm wondering here is the filter name - to be honest the variable isn't named that well because it's not JSON yet. I'm thinking `altis.search.autosuggest_query`?